riscv/include/mach_acpi.h

```diff
@@ -0,0 +1,27 @@
+/* acpi.h */
```

Comment:
Probably when you did the squeleton following amd64 code for building it you just included it.